src/composables/fungibleTokens.ts

```diff
@@ -48,6 +50,33 @@ const tokenBalances = useStorageRef<ITokenBalance[]>(
   STORAGE_KEYS.fungibleTokenBalances,
 );
 
+/**
+ * List of custom fungible tokens available on user's protocols.
```

Comment:
I suggest to update the comment as this is not a list of "custom" but "all" tokens.